GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -714,6 +719,20 @@ void GPURecoWorkflowSpec::run(ProcessingContext& pc)
 
   mTracker->Clear(false);
 
+  if (debugTFDump && mNDebugDumps < mConfParam->dumpBadTFs) {
+    mNDebugDumps++;
+    std::string filename = std::string("tpc_dump_") + std::to_string(pc.services().get<const o2::framework::DeviceSpec>().inputTimesliceId) + "_" + std::to_string(mNDebugDumps) + ".dump";
+    FILE* fp = fopen(filename.c_str(), "w+b");
+    std::vector<InputSpec> filter = {{"check", ConcreteDataTypeMatcher{gDataOriginTPC, "RAWDATA"}, Lifetime::Timeframe}};
+    for (auto const& ref : InputRecordWalker(pc.inputs(), filter)) {
+      auto data = pc.inputs().get<gsl::span<char>>(ref);
+      unsigned long size = data.size();
+      fwrite(&size, 1, sizeof(size), fp);
```

Comment:
@davidrohr this will make the dump unreadable via raw-file-reader, are you sure you need this?